ci-operator/config/redhat-developer/helm/redhat-developer-helm-release-3.11__helm-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,48 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+binary_build_commands: make -f openshift-ci/Makefile build
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tests
+images:
+- dockerfile_path: openshift-ci/Dockerfile.tests
+  from: base
+  to: helm
+releases:
```

Comment:
The other interop testing program configs are using the nightly latest build using this syntax.

```
releases:
  latest:
    candidate:
      product: ocp
      stream: nightly
      version: "4.14"
```

Is there is a specific reason that we are using `initial and latest` and things like `include_built_images: true` could you provide the reason(s)?

Reply:
No, i think we can use the above config for using nightly ocp releases